java/trace/src/main/scala/org/typelevel/otel4s/java/trace/WrappedSpanContext.scala

```diff
@@ -24,70 +24,37 @@ import org.typelevel.otel4s.trace.TraceFlags
 import org.typelevel.otel4s.trace.TraceState
 import scodec.bits.ByteVector
 
-private[java] final case class WrappedSpanContext(
-    jSpanContext: JSpanContext
-) extends SpanContext {
+private[otel4s] object WrappedSpanContext {
 
-  lazy val traceId: ByteVector =
-    ByteVector(jSpanContext.getTraceIdBytes)
-
-  def traceIdHex: String =
-    jSpanContext.getTraceId
-
-  lazy val spanId: ByteVector =
-    ByteVector(jSpanContext.getSpanIdBytes)
-
-  def spanIdHex: String =
-    jSpanContext.getSpanId
-
-  lazy val traceFlags: TraceFlags =
-    TraceFlags.fromByte(jSpanContext.getTraceFlags.asByte)
-
-  lazy val traceState: TraceState = {
+  def wrap(context: JSpanContext): SpanContext = {
```

Comment:
Nit: is `wrap` the correct terminology anymore?

Reply:
We can rename `WrappedSpanContext`to `SpanContextConversion`?

```scala
object SpanContextConversion {
  def toJSpanContext(ctx: SpanContext): JSpanContext
  def toSpanContext(ctx: JSpanContext): SpanContext // or fromJSpanContext
}
```